src/components/ui/field/FieldSeparator.vue

```
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import type { HTMLAttributes } from "vue"
+import { cn } from "@/lib/utils"
+import { Separator } from '@/components/ui/separator'
```

ESLint

[error] Expected "`@/components/ui/separator`" to come before "`@/lib/utils`".

(perfectionist/sort-imports)